rdlog.c

```diff
@@ -151,7 +151,7 @@ void rd_hexdump (FILE *fp, const char *name, const void *ptr, size_t len) {
 			cof += sprintf(charen+cof, "%c",
 				      isprint(p[i]) ? p[i] : '.');
 		}
-		fprintf(fp, "%08x: %-48s %-16s\n",
+		fprintf(fp, "%08zu: %-48s %-16s\n",
```

Comment:
Nevermind, I misread this for librdkafka which has stricter portability requirements.